akka-docs/src/main/paradox/stream/stream-refs.md

```diff
@@ -145,11 +144,11 @@ The process of preparing and running a `SinkRef` powered distributed stream is s
 Stream refs utilise normal actor messaging for their trainsport, and as such provide the same level of basic delivery guarantees. Stream refs do extend the semantics somewhat by demand re-delivery and sequence fault detection; in other words:
 
 - messages are sent over actor remoting
-  - which relies on TCP (classic remoting, or artery tcp) or Aeron UDP for basic redelivery mechanisms
+    - which relies on TCP (classic remoting, or artery tcp) or Aeron UDP for basic redelivery mechanisms
```

Comment:
what showing all bullets at the same level

Reply:
thanks :)